### ISSUE TITLE
Error returned by Apicurio not compliant with Confluent Schema Registry Client in Databricks

### ISSUE THREAD

**Issue by jelmew** [Apicurio/apicurio-registry#1081]
Hi all,

We are trying to use Apicurio as a schema registry. Apicurio should be fully compliant with Confluent's API. However, we are trying to register a schema from Azure databricks. The confluent client in this case gets (for unknown reasons) an error returned. The error from apicurio contains the field "detail", crashing the schema registry client from confluent without us knowing what the possible cause is. Apicurio logging doesn't help either.   How is this fixable/error showable?
```
Job aborted due to stage failure: Task 3 in stage 35.0 failed 1 times, most recent failure: Lost task 3.0 in stage 35.0 (TID 143, 10.139.64.4, executor driver): org.spark_project.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException: Unrecognized field "detail" (class org.spark_project.confluent.kafka.schemaregistry.client.rest.entities.ErrorMessage), not marked as ignorable (2 known properties: "error_code", "message"])
 at [Source: (sun.net.www.protocol.http.HttpURLConnection$HttpInputStream); line: 1, column: 222] (through reference chain: org.spark_project.confluent.kafka.schemaregistry.client.rest.entities.ErrorMessage["detail"]); error code: 50005
```

**Comment by EricWittmann** [comment]
@carlesarnal Can you take a look at this?  Apparently we are returning an error bean with an extra property on it, and Confluent's client can't handle that. :)

@jelmew what versions of everything are you using?

**Comment by carlesarnal** [comment]
Hello @EricWittmann, @jelmew. This is supposed to be fixed in version 1.2.1.Final. @jelmew, could you please let us know what versions are you using so we can confirm if this is a bug in recent versions or not? Thanks

**Comment by jelmew** [comment]
Bug is not in recent version, Thanksalot.

**Comment by EricWittmann** [comment]
Thanks for following up on this and closing it.

**Comment by EricWittmann** [comment]
@carlesarnal Can you confirm that this is done?

**Comment by carlesarnal** [comment]
@EricWittmann this was fixed [here](https://github.com/Apicurio/apicurio-registry/commit/4d5065552c46908186d1e3dd744b23fb91e129f6#diff-780c6be108fc8de5023f8e66f8ca7bc481ddde49e6526c303f7802c64289aa7fR124) and released in version `1.2.1.Final`.
